descriptors/ModuleDescriptor-template.json

```diff
@@ -50,7 +50,8 @@
         },
         {
           "methods": ["GET"],
-          "pathPattern": "/eholdings/providers/{providerId}/packages"
+          "pathPattern": "/eholdings/providers/{providerId}/packages",
+          "permissionRequired": ["kb-ebsco.provider.packages.get"]
```

Comment:
There's a typo, it should be "permissionsRequired"

Reply:
Thanks, updated